app/components/QuestionRow/QuestionRow.jsx

```diff
@@ -58,75 +78,112 @@ const QuestionRow = (props) => {
     }
     const newPinStatusValue = question.is_pinned ? 'false' : 'true';
     const data = new FormData(pinForm.current);
-    data.set("action", ACTIONS.PINNIN);
-    data.set("questionId", question.question_id);
-    data.set("value", newPinStatusValue);
+    data.set('action', ACTIONS.PINNIN);
+    data.set('questionId', question.question_id);
+    data.set('value', newPinStatusValue);
 
-    submit(
-      data,
-      { method: "post", action: url, replace: true }
-    );
-  }
+    submit(data, { method: 'post', action: url });
```

Comment:
Can you please add replace: true to avoid the following behavior. https://github.com/wizeline/wize-q-remix/issues/118

Reply:
I applied the change to both commends, didn't mean to delete it from this line of code, nice catch!